pkg/virt-launcher/virtwrap/util/libvirt_helper_test.go

```diff
@@ -218,7 +218,7 @@ var _ = Describe("LibvirtHelper", func() {
 		v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 		domain := &api.Domain{}
 		c := &converter.ConverterContext{
-			Architecture:     runtime.GOARCH,
+			Architecture:     converter.NewArchConverter(runtime.GOARCH),
```

Comment:
Do you think we can move this test in order to keep the converter private / unexposed?

Reply:
I'd prefer to leave this for a follow-up in case.
I'm trying to limit the scope of the PR to the archconverter only.
Does it work for you?